package/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "brave-ui",
   "main": "index.js",
-  "version": "0.30.5",
+  "version": "0.31.0",
```

Comment:
We update this when we publish to npm, so this can be undone

Reply:
Done